.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
```

Comment:
Is this file should have license ?

as a side note, editorconfig is supported by Theia ?

Reply:
I don't know about license.
Theia support editor config https://github.com/theia-ide/theia/tree/master/packages/editorconfig

Reply:
Editorconfig support `#`as being comment so license should be something like
```
###############################################################
# Copyright (c) 2018 Red Hat, Inc.
#
# This program and the accompanying materials are made
# available under the terms of the Eclipse Public License 2.0
# which is available at https://www.eclipse.org/legal/epl-2.0/
#
# SPDX-License-Identifier: EPL-2.0
###############################################################

```